### DISCUSSION TITLE
Seeking Guidance and Assistance in Understanding the Project - Willing to Pay for Consultation

### DISCUSSION THREAD

**Discussion by MMSahin** [kyleskom/NBA-Machine-Learning-Sports-Betting#364] [General]
Hello there,

I'm keen on understanding this project but lack the necessary knowledge in both Python and machine learning to dissect it on my own. As a result, I'd like to request assistance on the project's discussion channel. I'm willing to compensate for the guidance provided.

Thank you.

**Comment by MMSahin** [comment]
I`ve joined the discord group. Thank you. 

But i think what i need is a 1-2 hours consultation of someone who understand the project good. for example i need to understand which data is used in which way to train the model. how can i test the accuracy etc..

**Comment by esedeberis** [comment]
anyone got a new invite?

**Reply by TanerMontaner** [reply]
I'd love one too please!

**Reply by willisj318** [reply]
Same!
